code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -360,3 +360,29 @@
 	var/damage = ((min_damage / 4) + (max_damage / 4)) / 2 // We expect you to have 4 functional limbs- if you have fewer you're probably not going to be so good at lifting
 
 	return ceil(damage * (ceil(athletics_level / 2)) * fitness_modifier * maxHealth)
+
+/mob/living/carbon/human/proc/item_heal(mob/living/carbon/human/human, mob/user, brute_heal, burn_heal, heal_message_brute, heal_message_burn, required_bodytype,)
+	var/obj/item/bodypart/affecting = human.get_bodypart(check_zone(user.zone_selected))
+	if (!affecting || !(affecting.bodytype == required_bodytype ))
+		to_chat(user, span_warning("[affecting] is already in good condition!"))
+		return FALSE
+
+	var/brute_damaged = affecting.brute_dam > 0
+	var/burn_damaged = affecting.burn_dam > 0
+
+	if(((brute_heal > 0) && (brute_damaged)) || ((burn_heal > 0) && (burn_damaged)))
+		if (affecting.heal_damage(brute_heal, burn_heal, required_bodytype))
+			human.update_damage_overlays()
+			var/message
+			if (brute_damaged && burn_damaged)
+				message = "[heal_message_brute] and [heal_message_burn] on"
+			else if (brute_damaged)
+				message = "[heal_message_brute] on"
+			else
+				message = "[heal_message_burn] on"
+
+			user.visible_message(span_notice("[user] fixes some of the [message] [human]'s [affecting.name]."), \
+				span_notice("You fix some of the [message] [human == user ? "your" : "[human]'s"] [affecting.name]."))
+			return TRUE
+	to_chat(user, span_warning("[affecting] is already in good condition!"))
+	return FALSE
```

Comment:
```suggestion
	var/nothing_to_heal = ((brute_heal <= 0) || (!brute_damaged)) && ((burn_heal <= 0) || (!burn_damaged))
	if (nothing_to_heal  || !affecting.heal_damage(brute_heal, burn_heal, required_bodytype))
		to_chat(user, span_notice("[affecting] is already in good condition!"))
		return FALSE
	human.update_damage_overlays()
	var/message
	if (brute_damaged && burn_damaged)
		message = "[heal_message_brute] and [heal_message_burn] on"
	else if (brute_damaged)
		message = "[heal_message_brute] on"
	else
		message = "[heal_message_burn] on"

	user.visible_message(span_notice("[user] fixes some of the [message] [human]'s [affecting.name]."), \
		span_notice("You fix some of the [message] [human == user ? "your" : "[human]'s"] [affecting.name]."))
	return TRUE
```
Returning early means we don't have to indent this whole huge block of code

Reply:
We should also use `span_notice` because learning that your friend is in good health is good news, not bad.

Reply:
should probably use span_alert imo, notice is for the successful healing message, alert would be for a failed healing message

Reply:
This broke my code, Burn message no longer works :/ pls fix